core/db_classes/EE_Line_Item.class.php

```diff
@@ -1430,12 +1721,17 @@ protected function _get_descendants_of_type($type)
      * @deprecated 4.6.0
      * @param string $type like one of the EEM_Line_Item::type_*
      * @return EE_Line_Item
+     * @throws EE_Error
+     * @throws InvalidArgumentException
+     * @throws InvalidDataTypeException
+     * @throws InvalidInterfaceException
+     * @throws ReflectionException
      */
     public function get_nearest_descendant_of_type($type)
     {
         EE_Error::doing_it_wrong(
             'EE_Line_Item::get_nearest_descendant_of_type()',
-            __('Method replaced with EEH_Line_Item::get_nearest_descendant_of_type()', 'event_espresso'),
+            esc_html__('Method replaced with EEH_Line_Item::get_nearest_descendant_of_type()', 'event_espresso'),
```

Comment:
The class::method_name probably should not be localized here.

Reply:
will change